hackathon_site/dashboard/frontend/src/slices/order/adminOrderSlice.test.ts

```diff
@@ -0,0 +1,90 @@
+import store, { makeStore } from "slices/store";
+import {
+    initialState,
+    AdminOrderState,
+    adminOrderReducerName,
+    adminOrderSliceSelector,
+    isLoadingSelector,
+    errorSelector,
+    getOrdersWithFilters,
+    adminOrderSelectors,
+} from "slices/order/adminOrderSlice";
+import { get } from "api/api";
+import { makeMockApiListResponse } from "testing/utils";
+import { mockPendingOrders } from "testing/mockData";
+import { waitFor } from "@testing-library/react";
+
+jest.mock("api/api", () => ({
+    ...jest.requireActual("api/api"),
+    get: jest.fn(),
+}));
+
+const mockedGet = get as jest.MockedFunction<typeof get>;
+
+const mockStateWithAdminOrders = (adminOrderState?: Partial<AdminOrderState>) => ({
+    ...store.getState(),
+    [adminOrderReducerName]: {
+        ...initialState,
+        ...adminOrderState,
+    },
+});
+
+describe("Selectors", () => {
```

Comment:
It would be nice idea to make this name become specific. `adminOrderSlidesSelectors`. This name is same has been used for several other test files. It would be confusing if we are using PyCharm to run the different tests with the same name. 
<img width="511" alt="CleanShot 2023-01-08 at 02 11 47@2x" src="https://user-images.githubusercontent.com/36619969/211184802-75e36aae-4db7-4dd2-a38d-814663a3ff28.png">

Reply:
I only see this selector being used in this test file on my end? What other files is this name being used?

Reply:
<img width="796" alt="CleanShot 2023-01-09 at 17 43 00@2x" src="https://user-images.githubusercontent.com/36619969/211422996-f603eab2-d319-48bb-adf0-5cec203b28b7.png">

Reply:
Oh I get it, I'll get that done :)